### DISCUSSION TITLE
Why not version 7.8.0

### DISCUSSION THREAD

**Discussion by joenzx** [apache/skywalking#8045] [General]
Deployed Skywalking on K8S, tested versions 7.5 and 7.6, and was able to access the home page, but when the image was changed to 7.8, the home page was inaccessible。

apiVersion: v1
kind: ConfigMap
metadata:
  name: skywalking-cm
data:
  SW_STORAG: 'elasticsearch7'
  SW_STORAGE_ES_CLUSTER_NODES: 'localhost:9200'
  SW_ES_USER: 'elastic'
  SW_ES_PASSWORD: 'elastic'
  SW_CORE_GRPC_PORT: '11800'
  SW_CORE_REST_PORT: '12800'

deployment-oap.yaml
`
apiVersion: apps/v1
kind: Deployment
metadata:
  labels:
    app: skywalking
  name: skywalking
  namespace: default
spec:
  replicas: 2
  selector:
    matchLabels:
      app: skywalking
  template:
    metadata:
      labels:
        app: skywalking
    spec:
      containers:
        - envFrom:
          - prefix: SW_
            configMapRef: 
              name: skywalking-cm                  
          image: apache/skywalking-oap-server:8.7.0-es7
          imagePullPolicy: IfNotPresent         
          name: skywalking
          ports:
            - containerPort: 12800
              name: http
              protocol: TCP
            - containerPort: 11800
              name: grpc
              protocol: TCP
          resources:
            limits:
              cpu: '2'
              memory: 2Gi
            requests:
              cpu: '1'
              memory: 2Gi

`

service-oap.yaml
`
apiVersion: v1
kind: Service
metadata:
  name: skywalking
  labels:
    app: skywalking
spec:
  type: ClusterIP
  ports:
    - name: http
      port: 12800
      protocol: TCP
      targetPort: 12800
    - name: grpc
      port: 11800
      protocol: TCP
      targetPort: 11800
  selector:
    app: skywalking

`
ui.yaml
`
apiVersion: apps/v1
kind: Deployment
metadata:
  labels:
    app: skywalking-ui
  name: skywalking-ui
  namespace: default
spec:
  replicas: 1
  selector:
    matchLabels:
      app: skywalking-ui
  template:
    metadata:
      labels:
        app: skywalking-ui
    spec:
      containers:
        - env:
            - name: SW_OAP_ADDRESS
              value: "skywalking:12800"          
          image: apache/skywalking-ui:8.5.0
          imagePullPolicy: IfNotPresent         
          name: skywalking-ui
          ports:
            - containerPort: 8080
              name: http
              protocol: TCP
          resources:
            limits:
              cpu: '2'
              memory: 1Gi
            requests:
              cpu: '1'
              memory: 1Gi

`
ui-service.yaml

`
apiVersion: v1
kind: Service
metadata:
  name: skywalking-ui
  labels:
    app: skywalking-ui
spec:
  type: ClusterIP
  ports:
    - name: http
      port: 8080
      protocol: TCP
      targetPort: 8080
  selector:
    app: skywalking-ui

`

If the image above is 7.5 or 7.6, it will run fine.
If the version is 7.8, the page cannot be accessed. The following error message is displayed
![45336c4773bc2be9e37366dda8343de](https://user-images.githubusercontent.com/42016493/139651378-f4ff834e-8632-49ea-9590-dd9dad01f0a2.png)

**Comment by wu-sheng** [comment]
What you posted has nothing relating to ElasticSearch server.

**Reply by joenzx** [reply]
> What you posted has nothing relating to ElasticSearch server.

I don't know the exact reason. The reason is that I can directly access the home page using Apache/Skywalking - OAP-server: 8.6.0-ES7. Other configurations remain unchanged

**Reply by wu-sheng** [reply]
Check the doc, rather than following some unknown version blogs. 
We changed webapp container from Zuul to Spring Gateway v2. So a http:// prefix is required.
